src/ol/interaction/pointer.js

```diff
@@ -187,8 +187,8 @@ ol.interaction.Pointer.handleEvent = function(mapBrowserEvent) {
     if (mapBrowserEvent.type == ol.MapBrowserEventType.POINTERDRAG) {
       this.handleDragEvent_(mapBrowserEvent);
     } else if (mapBrowserEvent.type == ol.MapBrowserEventType.POINTERUP) {
-      this.handleUpEvent_(mapBrowserEvent);
-      this.handlingDownUpSequence = false;
+      this.handlingDownUpSequence = this.handleUpEvent_(mapBrowserEvent) &&
+          this.targetPointers.length > 0;
```

Comment:
@tchandelle can you rework this to avoid having side effects in the logical expression?  E.g.

```
var unhandledUp = this.handleUpEvent_(mapBrowserEvent);
this.handlingDownUpSequence = unhandledUp && this.targetPointers.length > 0;
```